include/tscore/ConsistentHash.h

```diff
@@ -46,17 +47,17 @@ typedef std::map<uint64_t, ATSConsistentHashNode *>::iterator ATSConsistentHashI
  */
 
 struct ATSConsistentHash {
```

Comment:
As long as you're in here, why is this a struct not a class?

Reply:
You'd have to ask @jrushford - I wanted to minimize the changes in this area, it's really outside the scope of the PR.